karma.shared.js

```diff
@@ -2,16 +2,19 @@
 
 /* This is the shared karma config */
 
+var merge = require('lodash.merge');
 var RewirePlugin = require('rewire-webpack');
-var webpackConfig = require('./webpack.config.js');
+var webpackConfig = require('./webpack.config');
 
+
+var newWebpackConfig = merge({}, webpackConfig);
```

Comment:
this works but we also have the [object-assign](https://github.com/mozilla/payments-ui/blob/master/package.json#L16) polyfill which is used elsewhere like `assign({}, ...)`